CMakeLists.txt

```diff
@@ -117,6 +117,24 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
+if (UNIX)
```

Comment:
PLATFORM_MAC?

Reply:
UNIX will be true on both linux and mac, or would you prefer your custom variables to be used instead?

Reply:
Custom variables are better because it explicitly indicates that we do not want to apply it for iOS and Android (which are unix-like systems).

We do not change instruments for mobile platforms without complex research & tests.

Reply:
understood. changed.